modules/sdk/project-templates/project-templates-theme/src/main/resources/archetype-resources/build.gradle

```diff
@@ -14,9 +14,9 @@ apply plugin: "com.liferay.portal.tools.theme.builder"
 
 dependencies {
 #if (${liferayVersion.startsWith("7.0")})
-	cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.1.3"
+	cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.0.0"
 #elseif (${liferayVersion.startsWith("7.1")} || ${liferayVersion.startsWith("7.2")} || ${liferayVersion.startsWith("7.3")})
-	cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.1.3"
+	cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.0.3"
```

Comment:
you can not change it. the 3.1.3 is not fit for 7.1 7.2 and 7.3. It will cause the three versions theme build failed.

Reply:
@lawrence-lee we can not directly use 3.1.3 for 7.1 7.2 and 7.3 although it fix jdk 11 issue. Maybe we need to make 3.1.3 to support these three versions, otherwise we can not use this version for  7.1 7.2 and 7.3.

Reply:
Makes sense, thanks @simonjhy